### PR TITLE
Ease log dependency

### LINKED ISSUE #113: consider degrading log version from 0.4 to 0.3
It shows Unknown for the file name in TiKV log, and hard to debug for us.

### PR DESCRIPTION
**Pull request by Hoverbear**
Fixes #113

### CODE REVIEW

**Comment by breeswish** [comment]
Oh. >0.3 is a good idea!

File: Cargo.toml [hunk 1]
```diff
@@ -12,7 +12,7 @@ description = "The rust language implementation of Raft algorithm."
 categories = ["algorithms", "database-implementations"]
 
 [dependencies]
-log = "0.4"
+log = ">0.3"
```
[the remark this answers is not in the document]
Comment (Hoverbear) [reply Cargo.toml]: :+1:

**Comment by BusyJay** [comment]
So if log 0.5 breaks again, we need another PR to fix it? 🤣

**Comment by breeswish** [comment]
@BusyJay we don't need to publish a new version if log 0.5 published, as long as info!, trace! and so on interfaces are not changed.

**Pull request merged**